GUI for roles added & /lg roles update & setFoodLevel fix

Commantary opened the pull request:
- Il y a un GUI pour changer les rôles plus facilement. L'item pour le GUI est donner si le joueur a la perm 'loupgarou.admin'.
- Il y a un item pour lancer la partie simplement.
- Changement de l'event 'FoodLevelEvent' qui mettait la nourriture à 6, et c'est franchement pas utile donc il est à 20.

Commantary committed:
GUI for roles and /lg roles update

- Il y a un GUI pour changer les rôles plus facilement. L'item pour le GUI est donner si le joueur a la perm 'loupgarou.admin'.
- Il y a un item pour lancer la partie simplement.
- Changement de l'event 'FoodLevelEvent' qui mettait la nourriture à 6, et c'est franchement pas utile donc il est à 20.

leomelki reviewed (changes requested):
Le food level doit rester à 6 pour éviter que le joueur sprint. Sinon, il aura des rollback une fois sur la plaque.
Dans la classe MainLg, tu as ajouté beaucoup de if inutiles et il ne faut pas reformatter le code, sinon on ne comprend rien au commit :/

Commantary committed:
Revert "GUI for roles and /lg roles update"

This reverts commit 196b94c85a191baae688572085316ea6b1627905.

Commantary committed:
GUI for roles and /lg roles update (fix space changes)

- Il y a un GUI pour changer les rôles plus facilement. L'item pour le GUI est donner si le joueur a la perm 'loupgarou.admin'.
- Il y a un item pour lancer la partie simplement.

Commantary commented:
J'ai annuler les changements pour la food ainsi que le reformatage du code qui s'était fait auto par IntelliJ.
J'ai essayer d'enlever 1-2 if, j'ai fais de mon mieux :)

Commantary committed:
fix d'un oublie

J'ai oublier de mettre le registerEvent pour PlayerInteractListener

Commantary committed:
Revert "fix d'un oublie"

This reverts commit 585abebb3086340891584a4d39490a18fae646e3.

Commantary committed:
fix d'un oublie

J'ai oublier de mettre le registerEvent pour PlayerInteractListener

Commantary committed:
Fix items disappears

Les items disparaîssent quand on valide les rôles ou en faisant '/lg joinall'

TheOptimisticFactory commented:
Il y a quelques erreurs de compilation corrigeables avec ce diff: https://github.com/TheOptimisticFactory/LoupGarou/commit/7df04392ecb443d42207b859fcbbf4188e8080ae
